### PR TITLE
Create TestNull3

### PR DESCRIPTION
**Pull request by nehaa2188**

### CODE REVIEW

File: TestNull2 [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ 
+  public static void main(String[] args) {
+    //create object of LinkedHashSet
+    LinkedHashSet lhashSet = new LinkedHashSet();
+   
+    /*
+      Add an Object to LinkedHashSet using
+      boolean add(Object obj) method of Java LinkedHashSet class.
+      This method adds an element to LinkedHashSet if it is not
+      already present in LinkedHashSet.
```
Comment (nehaa2188) [inline TestNull2]: screen_shot_2017-09-05_at_7_06_00_pm
Happening due to the long list of apps/actions

File: TestNull2 [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import java.util.LinkedHashSet;
+ 
+public class SimpleLinkedHashSetExample {
+
```
Comment (nehaa2188) [inline TestNull2]: screen_shot_2017-09-05_at_7_06_00_pm
Happening due to the long list of apps/actions

Comment (nehaa2188) [reply TestNull2]: test

**Comment by nehaa2188** [comment]
test

**Comment by nehaa2188** [comment]
foo bar baz
fsdfds
foo bar baz
fsdfds

**Comment by nehaa2188** [comment]
foo bar baz
fsdfds
foo bar baz
fsdfds
foo bar baz
fsdfds
foo bar baz
fsdfds
foo bar baz
fsdfds
foo bar baz
fsdfds

**Comment by nehaa2188** [comment]
public class URLExpSimple {

    
    public static void main(String[] args) {
        try {
            URL mySite = new URL("http://www.cs.utexas.edu/~scottm");
            URLConnection yc = mySite.openConnection();
            Scanner in = new Scanner(new InputStreamReader(yc.getInputStream()));
            int count = 0;
            while (in.hasNext()) {
                System.out.println(in.next());
                count++;
            }
            System.out.println("Number of tokens: " + count);
            in.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
public class URLExpSimple {

    
    public static void main(String[] args) {
        try {
            URL mySite = new URL("http://www.cs.utexas.edu/~scottm");
            URLConnection yc = mySite.openConnection();
            Scanner in = new Scanner(new InputStreamReader(yc.getInputStream()));
            int count = 0;
            while (in.hasNext()) {
                System.out.println(in.next());
                count++;
            }
            System.out.println("Number of tokens: " + count);
            in.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
public class URLExpSimple {

    
    public static void main(String[] args) {
        try {
            URL mySite = new URL("http://www.cs.utexas.edu/~scottm");
            URLConnection yc = mySite.openConnection();
            Scanner in = new Scanner(new InputStreamReader(yc.getInputStream()));
            int count = 0;
            while (in.hasNext()) {
                System.out.println(in.next());
                count++;
            }
            System.out.println("Number of tokens: " + count);
            in.close();
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

**Comment by GitTester2108** [comment]
@nehaa2188 check this

**Comment by GitTester2108** [comment]
GTM

**Comment by nehaa2188** [comment]
Test

**Comment by nehaa2188** [comment]
GTM

**Comment by nehaa2188** [comment]
Grm

**Comment by nehaa2188** [comment]
Gtm

**Pull request merged**